.github/workflows/wheels.yml

```
@@ -219,3 +221,42 @@ jobs:
           source ci/upload_wheels.sh
           set_upload_vars
           upload_wheels
+
+  publish:
+    if: >
+      github.repository == 'pandas-dev/pandas' &&
+      github.event_name == 'push' &&

```

```suggestion
      github.event_name == 'release' &&
```

To match documentation